scripts/train/train.py

```diff
@@ -294,6 +294,14 @@ def main(cfg):
 
     # Build the Trainer
     print('Building trainer...')
+    save_folder = cfg.get('save_folder', None)
+    save_latest_filename = cfg.get('save_latest_filename',
+                                   'latest-rank{rank}.pt')
+    save_overwrite = cfg.get('save_overwrite', False)
+    save_weights_only = cfg.get('save_weights_only', False)
+    autoresume_default = False
```

Comment:
Btw I don't think this actually configures `autoresume` by default on. If and only if all requirements are configured properly, then we turn on `autoresume`. Shouldn't we emit a warning that `autoresume` is not turned on by default if one of these requirements is not met?

Reply:
Yea, I think this is the closest we can get. All of these are on by default except save folder, which users need to do autoresume / always provide.

I think it's ok to not emit a warning. These are rarely touched, and this is a second line of defense -- normally all of our configs have autoresume on anyways. This will trigger on >95% of user workloads, so I'm not really concerned about it